<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forgetpasswduiClass</class>
 <widget class="QDialog" name="forgetpasswduiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>忘记密码</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/res.qrc">
    <normaloff>:/res/default_user_icon.png</normaloff>:/res/default_user_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="9" column="1">
      <widget class="TimerButton" name="pushButton_get_varifiycode_forget">
       <property name="text">
        <string>获取验证码</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <widget class="InputLine" name="lineEdit_id_binded_email"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>账号绑定邮箱：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="InputLine" name="lineEdit_user_account_forget"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>账号：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="InputLine" name="lineEdit_passwd_forget"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>密码：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="InputLine" name="lineEdit_varifiy_passwd_forget"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>验证密码：</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="pushButton_submit_the_change_passwod_forget">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEdit_varifiy_code_forget"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>验证码：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TimerButton</class>
   <extends>QPushButton</extends>
   <header location="global">timerbutton.h</header>
  </customwidget>
  <customwidget>
   <class>InputLine</class>
   <extends>QLineEdit</extends>
   <header location="global">inputline.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
